src/ActionsMenuItem.scss

```diff
@@ -0,0 +1,30 @@
+.ActionsMenuItem__button {
+    width: 4rem;
+    float: right;
+    border: 0;
+    background-color: inherit;
+}
+
```

Comment:
Let's add style for the checkboxes to be vertically centred.
`
.ActionsMenuItem__checkbox {
    margin: auto 0;
}
`